frontend/app/admin/advertisement/create/page.tsx

```diff
@@ -0,0 +1,303 @@
+              border="1px"
+              borderColor="gray.400"
+            />
+            <FormErrorMessage>
+              {
+                errors.find((e) => {
+                  return e.path[0] === 'name'
+                })?.message
+              }
+            </FormErrorMessage>
+          </FormControl>
+          <FormControl
+            isInvalid={
+              !!errors.find((e) => {
+                return e.path[0] === 'imageUrl'
+              })
+            }
+          >
+            <FormLabel>
+              <Text>画像</Text>
+            </FormLabel>
+            {/* <Input
+              bg={THEME_COLOR.SECONDARY_WHITE}
+              type="file"
+              accept="image/*"
+              value={imageUrl}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setImageUrl(e.target.value)
+              }
+            /> */}
+            {/* TODO: 画像をs3に渡せるように追加対応を行う
+            <>
+              <img src="" />
+            </> */}
+            <FormErrorMessage>
+              {
+                errors.find((e) => {
+                  return e.path[0] === 'imageUrl'
+                })?.message
+              }
+            </FormErrorMessage>
+          </FormControl>
+          <FormControl
+            isInvalid={
+              !!errors.find((e) => {
+                return e.path[0] === 'author'
+              })
+            }
+          >
+            <FormLabel>企業名</FormLabel>
+            <Input
+              bg={THEME_COLOR.SECONDARY_WHITE}
+              type="text"
+              value={author}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setAuthor(e.target.value)
+              }
+              aria-required={true}
+              border="1px"
+              borderColor="gray.400"
+            />
+            <FormErrorMessage>
+              {
+                errors.find((e) => {
+                  return e.path[0] === 'author'
+                })?.message
+              }
+            </FormErrorMessage>
+          </FormControl>
+          <FormControl
+            isInvalid={
+              !!errors.find((e) => {
+                return e.path[0] === 'url'
+              })
+            }
+          >
+            <FormLabel>
+              <Text>リンク</Text>
+            </FormLabel>
+            <Input
+              bg={THEME_COLOR.SECONDARY_WHITE}
+              type="text"
+              value={url}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+                setUrl(e.target.value)
+              }
+              border="1px"
+              borderColor="gray.400"
+            ></Input>
+            <FormErrorMessage>
+              {
+                errors.find((e) => {
+                  return e.path[0] === 'url'
+                })?.message
+              }
+            </FormErrorMessage>
+          </FormControl>
+          <Flex justify="center" align="center" gap={'10px'}>
```

Comment:
```suggestion
          <Flex justify="center" align="center" gap={'10px'} height={100px}>
```

みたいに高さをあらかじめ持たせればカクツキ無くなったりしないですかね？
高さはデザイン意識して調整してもらいたいです、